xarray/core/common.py

```diff
@@ -346,6 +346,28 @@ def assign_coords(self, **kwargs):
         data.coords.update(results)
         return data
 
+    def assign_attrs(self, *args, **kwargs):
+        """Assign new attrs to this object, returning a new object equivalent
```

Comment:
I'm sorry, I don't follow. I tried to match the conventions of the rest of the file. Do you mean this:

```
def assign_attrs(self, *args, **kwargs):
    """
    Assign new attrs to this object, returning a new object equivalent
    to...
```

or

```
def assign_attrs(self, *args, **kwargs):
    """Assign new attrs to this object.

    Returns a new object equivalent to...
```